### ISSUE TITLE
Porting this plugin to craft 3

### ISSUE THREAD

**Issue by pixeldeluxe** [mmikkel/CpSortableCustomColumns-Craft#13]
Hi,

I'd love to pay a fee for the plugin if it is ported, so please ;-)
Keep up the good work.

Cheers,
Martijn

**Comment by mmikkel** [comment]
Thanks @pixeldeluxe – I'm not really in the paid plugin game, though :)

**Comment by modelesque** [comment]
@mmikkel do you plan to port this to Craft 3?

**Comment by mmikkel** [comment]
@rswhite7 No, sorry. Doesn't mean it won't happen, but it's unlikely I'll get to it. The plugin is MIT though, so anyone could do it if they wanted to :)
